test/Microsoft.AspNetCore.Razor.Language.Test/Legacy/HtmlAttributeTest.cs

```diff
@@ -219,7 +224,7 @@ public void UnquotedAttributeWithCodeWithSpacesInDocument()
             ParseDocumentTest("<input value=@foo />");
         }
 
-        [Fact]
+        [Fact(Skip = "ConditionalAttributeCollapser doesn't exist in the new tree")]
```

Comment:
If these tests aren't meaningful should we delete them?

I don't see much value in keeping them around if we don't plan to ever re-enable them. Generally we only want to skip tests if there's a blocking bug that will get fixed.

Reply:
Please log an issue and put that in the skip message. This is our way to tracking future work 😆